lib/beacon/broadcast.go

```diff
@@ -47,7 +47,6 @@ func writeBroadcasts(ctx context.Context, inbox <-chan []byte, port int) error {
 		intfs, err := net.Interfaces()
 		if err != nil {
 			l.Debugln(err)
-			return err
```

Comment:
`net.Interfaces()` will throw an error on Android. We can still carry on and use the fallback broadcast address `255.255.255.255` later on.

Reply:
This comment should maybe live directly in the code.